src/js/app-es6/controllers/PontoController.js

```diff
@@ -56,32 +56,52 @@ class PontoController {
 
         let msg;
 
-            if (this._listaPontos.pontos.length == 0) {
-                msg = 'Lista já está vazia'
+        if (this._listaPontos.pontos.length == 0) {
+            msg = 'Lista já está vazia'
+            this._mensagem.toast = msg;
+        } else {
+            if (!this._listaPontos.esvazia(this._listaPontos)) {
```

Comment:
Unexpected if as the only statement in an else block.

[Read more about it here.](http://eslint.org/docs/rules/no-lonely-if)